specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2023-04-01/mfe.json

```diff
@@ -1,14 +1,3416 @@
 {
   "swagger": "2.0",
   "info": {
-    "title": "Azure Machine Learning Workspaces",
+    "title": "Azure Machine Learning Services",
```

Comment:
this change is making the Go SDK generation to fail.
Could we revert this on this PR and introduce this change in a clean PR and resolve any issues related with this on that PR?

Reply:
created alignment between RPs to fix this